splunk_add_on_ucc_framework/tabs/__init__.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2024 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from typing import Dict, Any
+
+from splunk_add_on_ucc_framework.tabs.logging_tab import LoggingTab
+from splunk_add_on_ucc_framework.tabs.tab import Tab
+
+
+TAB_TYPES = [
+    LoggingTab,
+]
+
+
+def resolve_tab(definition: Dict[Any, Any]) -> Tab:
```

Comment:
Can you please add a bit of a description for `definition`?

Reply:
In docstring? Sure.